ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -1197,7 +1197,7 @@ private void finalizeMetadataChunk() {
           new BlobProperties(getBlobSize(), passedInBlobProperties.getServiceId(), passedInBlobProperties.getOwnerId(),
               passedInBlobProperties.getContentType(), passedInBlobProperties.isPrivate(),
               passedInBlobProperties.getTimeToLiveInSeconds(), passedInBlobProperties.getCreationTimeInMs(),
-              passedInBlobProperties.getAccountId(), passedInBlobProperties.getContainerId());
+              passedInBlobProperties.getAccountId(), passedInBlobProperties.getContainerId(), false);
```

Comment:
wouldn't setting this to `false` cause a correctness issue? `GetBlobInfo` relies on getting and returning the `BlobProperties` of the metadata chunk. So shouldn't this be `passedInBlobProperties.isEncrypted()`?

Reply:
Yes - this was left this way because in my mind this should be done as part of the router patch. It is not immediately clear to me who would create the `BlobProperties` with the encryption field set - whether it will be part of the passed in properties or whether router will figure it out some other way and create its own `BlobProperties`.

Now made the change to use `isEncrypted` from the passed in properties.